evalml/tests/automl_tests/test_automl.py

```diff
@@ -1120,16 +1122,22 @@ def test_percent_better_than_baseline_in_rankings(objective, pipeline_scores, ba
     # Ok to only use binary labels since score and fit methods are mocked
     X, y = X_y_binary
 
+    objective_type = ProblemTypes.BINARY
+    if isinstance(objective, RegressionObjective):
+        objective_type = ProblemTypes.REGRESSION
+    elif isinstance(objective, MulticlassClassificationObjective):
+        objective_type = ProblemTypes.MULTICLASS
```

Comment:
This is fine for now. But once we add objectives which aren't specific to one problem type (like timeseries), we'll have to remember to change this. Is there anything we can do to future-proof this test?

One option: add `problem_type` to the `parametrize`. Then add
```python
if not objective.is_defined_for_problem_type(problem_type):
    pytest.skip()
```
(syntax prob wrong)

No prob if you don't get to this. I hope that in the wild, we'll always know the `problem_type` when we're working with objectives, so this doesn't worry me outside of our tests.

Reply:
Made this change! Good point - it does make the test simpler! And when we add time series to automl, we just need to update the list of values that's parametrized so that's great.